docs/data/dataset.html

```diff
@@ -1,936 +0,0 @@
-<?js
```

Comment:
Please don't remove the old docs without saving the content! 
I see two ways of saving the content:
1. Keep the old docs and provide the newly generated docs as addition.
2. transfer the old content to the jsdocs and/or the Readme.

Reply:
The old docs seem broken, I get redirection loop when I open index.html, 404s when I open some file directly and follow a link.

Reply:
Can you please review and merge #2 This fixes the old docs.

Reply:
(By the way: I added you as maintainer for this, and all the other projects. Pleaso ALWAYS create a pull request for every now feature and fix. Thanks!)

Reply:
So, the old docs are still there and the new docs are in tsdocs folder.